api/test/integration/run_tests.py

```diff
@@ -187,14 +158,8 @@ def get_results(filename: str = None):
         calculate_result(filename)
     else:
         os.chdir(RESULTS_FOLDER)
-        for file in sorted(glob.glob('test_*_standalone*')):
-            print(f"{file} - Standalone environment")
-            calculate_result(file)
-        for file in sorted(glob.glob('test_*_cluster*')):
-            print(f"{file} - Cluster environment")
-            calculate_result(file)
-        for file in sorted(glob.glob('test_rbac*')):
-            print(file)
+        for file in sorted(glob.glob('test_*')):
+            print(f"{file}")
```

Comment:
No need to use f-string
```suggestion
            print(file)
```